brain/circuit.h

```diff
@@ -209,14 +217,19 @@ class Circuit
         const GIDSet& preGIDs, const GIDSet& postGIDs,
         SynapsePrefetch prefetch = SynapsePrefetch::none) const;
 
+    /**
+     * Return the uri where morphology meshes for this circuit can be located.
+     */
+    BRAIN_API URI getMeshSource() const;
```

Comment:
this looks really lost here. No other source paths are exposed here. Also, this is beyond the 'specification' of the circuit config to know about a mesh path. This shouldn't be in the API imo.

Reply:
I'll remove it, but I'll have to do some gymnastics to adapt the depending code.